str/associatr/helper/cell_per_individual_extractor.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+"""
+This script aims to extract the number of cells per individual per cell type from the anndata object (containing all individuals, all chromosomes).
+The output is a CSV file with three columns: CPG ID, cell type, counts of cells.
+
+analysis-runner --dataset "bioheart" --access-level "test" --description "get cell counts per individual" --output-dir "str/associatr/helper" --image australia-southeast1-docker.pkg.dev/cpg-common/images/scanpy:1.9.3 \
+--storage=10G --cpu=4 \
```

Comment:
```suggestion
--storage=10G \
```
python is single threaded, so probably no benefit here